mobile-app/App.tsx

```diff
@@ -1,30 +1,32 @@
 import * as React from 'react';
+import { useState } from 'react';
 import { StyleSheet, View, ScrollView } from 'react-native';
+import { initNewGameState } from './api-client/LocalplayService';
 import Board from './components/Board/Board';
 import { GameState } from './utils';
 
-const gamestate: GameState = {
-  turn: 'white',
-  winner: null,
-  winningPath: null,
-  board: [
-    [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
-    [{ value: 'empty' }, { value: 'empty' }, { value: 'white' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
-    [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'black' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
-    [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
-    [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
-    [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'black' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
-    [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
-    [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'white' }, { value: 'empty' }, { value: 'empty' }],
-    [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
-  ],
-}
+const gameStateInitialValue: GameState = {
+  board: [],
+  turn: "white",
+  winner: null, 
+  winningPath: null
+};
 
 export default function App() {
+  const [gameState, setGameState] = useState<GameState>(gameStateInitialValue);
```

Comment:
I don't understand why a default value is needed, because we want to render the component after the result of our api call. But it's seems to be the way to go with React hooks.

Reply:
Today the problem is, this is the home page of our app, and we can't afford to wait on the API to answer something to display our home page. But once https://github.com/marmelab/Hex/pull/42 will have been merged, this will no longer be a problem and we'll be able to get rid of this initial value 🙂

Reply:
Ok, I get it thx !